tests/app/main/test_views.py

```diff
@@ -6,7 +6,7 @@
     from io import BytesIO as StringIO
 
 import mock
-from dmutils.apiclient import APIError
+from dmutils.apiclient import HTTPError, REQUEST_ERROR_MESSAGE
```

Comment:
I have always been told not to use constants from the code you're testing in tests - the tests should have the expected values in the test package (the idea being that if someone messes up the message in the main code then the test will fail because the message is wrong).